.github/workflows/ci.yml

```diff
@@ -60,3 +60,39 @@ jobs:
       - name: Check if the JDK index is up to date
         shell: bash
         run: sbt checkIndex
+  
+  download:
+    name: Download and Test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        distribution: [temurin, graalvm, openj9, corretto, oracle]
+        version: [8, 11, 17]
+        exclude:
+          - distribution: graalvm
+            version: 8
+          - distribution: oracle
+            version: 8
+          - distribution: oracle
+            version: 11
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout current branch
+        uses: actions/checkout@v2
+
+      - name: Download Java
+        id: download-java
+        uses: typelevel/download-java@v1
```

Comment:
Welp. I thought the `v1` thing worked automatically, but apparently you have to manually add a `v1` tag and you are expected to move it whenever you release a new version of your action.